src/main/java/com/cloudbees/groovy/cps/Builder.java

```diff
@@ -153,8 +153,8 @@ public Block sequence(Block b) {
         return b;
     }
 
-    public Block closure(int line, List<String> parameters, Block body) {
-        return new ClosureBlock(loc(line),parameters,body,closureType);
+    public Block closure(int line, List<Class> parameterTypes, List<String> parameters, Block body) {
```

Comment:
Not here. It was needed in `ClosureBlock` serialization form.

I'm going to write up the compatibility model of this module, since this is clearly not obvious to many.